Makefile

```
@@ -6,6 +6,8 @@
 
 NODE_BIN=$(CURDIR)/node_modules/.bin
 
+PACKAGE_NAME := edx_credentials_themes
```

@shadinaif this name is confusing because it's not a PyPi python package.

```suggestion
THEME_NAME := edx_credentials_themes
```

---

done